java/src/jmri/implementation/MultiIndexProgrammerFacade.java

```diff
@@ -255,18 +257,32 @@ synchronized public void writeCV(String CV, int val, jmri.ProgListener p) throws
 
     @Override
     synchronized public void readCV(String CV, jmri.ProgListener p) throws jmri.ProgrammerException {
+        readCV(CV, p, NO_DEFAULT_CV);
+    }
+
+    @Override
+    synchronized public void readCV(String CV, jmri.ProgListener p, int startVal) throws jmri.ProgrammerException {
         useProgrammer(p);
         parseCV(CV);
+        _startVal = startVal;
         if (valuePI == -1) {
             lastValuePI = -1;  // next indexed operation needs to write PI, SI
             lastValueSI = -1;
 
             state = ProgState.PROGRAMMING;
-            prog.readCV(_cv, this);
+            if (startVal == NO_DEFAULT_CV) {
+                prog.readCV(_cv, this);
+            } else {
+                prog.readCV(_cv, this, _startVal);
+            }
```

Comment:
You don't need this `if` statement since `prog.readCV(_cv, this, NO_DEFAULT_CV)` is the same as `prog.readCV(_cv, this)`.